metadata.json

```diff
@@ -29,7 +29,7 @@
     }
   ],
   "dependencies": [
-    { "name": "puppetlabs/concat", "version_requirement": ">= 1.1.0 <2.0.0" },
+    { "name": "puppetlabs/concat", "version_requirement": ">= 1.1.0 <=2.1.0" },
     { "name": "puppetlabs/stdlib", "version_requirement": ">=4.2.0 <5.0.0" },
```

Comment:
Since PL is pretty good with semver I think this should actually be <=3.0.0 -- sorry I didn't catch it before.